src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -119,4 +123,19 @@ public void moveArmToPosition() {
         .getPIDController()
         .setReference(armPositions.get(currentArmPosition), CANSparkMax.ControlType.kPosition);
   }
```

Comment:
`moveArmToPosition()` would be more helpful as `moveArmToPosition(double targetPosition)` or `moveArmToPosition(armPosition targetPosition)`. This would allow commands and other subsystem methods to choose where to go, and lets you get rid of `currentArmPosition`.